src/lambda_function/events/generate_test_event.py

```diff
@@ -0,0 +1,144 @@
+"""
+This is a utility script that generates a fake S3 event notification and writes
+it out as JSON. This test event can be used to test the Lambda which
+triggers the Glue pipeline.
+
+Two types of events can be generated:
+
+    1. An event with a single record (single-records.json).
+    2. An event with multiple records (records.json).
+
+See https://github.com/Sage-Bionetworks/recover/tree/main/src/lambda_function
+for detailed information how to use this event with the Lambda.
+"""
+
+import argparse
+import json
+import boto3
+
+SINGLE_RECORD_OUTFILE = 'single-record.json'
+MULTI_RECORD_OUTFILE = 'records.json'
+
+
+def read_args() -> argparse.Namespace:
+    parser = argparse.ArgumentParser(
+      description="Generate a json file of a mocked SQS event for testing.")
```

Comment:
Nit: Try this, then you can remove your "Default" strings
```suggestion
    parser = argparse.ArgumentParser(
      description="Generate a json file of a mocked SQS event for testing.",
      formatter_class=argparse.ArgumentDefaultsHelpFormatter
    )
```